app/src/ui/app.tsx

```diff
@@ -2613,6 +2615,7 @@ export class App extends React.Component<IAppProps, IAppState> {
           isShowingFoldout={this.state.currentFoldout !== null}
           aheadBehindStore={this.props.aheadBehindStore}
           commitSpellcheckEnabled={this.state.commitSpellcheckEnabled}
+          onCherryPick={this.startCherryPickWithoutBranch}
```

Comment:
That totally makes sense. However, I was thinking `startCherryPickWithoutBranch` and `startCherryPickWithBranch` as I liked it answering question of "do I need branch dialog"  in case I had the flow noted in my brainstorming about if user drag and dropped on to the menu dropdown without dropping on a branch. This would invoke the without branch but still be a drag and drop. 

I did not document that other possible scenario on `startCherryPickWithoutBranch`.. cause still undecided if that scenario makes sense..